src/awkward/contents/recordarray.py

```diff
@@ -26,6 +26,86 @@
 
 @final
 class RecordArray(Content):
+    """
+    RecordArray represents an array of tuples or records, all with the
+    same type. Its `contents` is an ordered list of arrays.
```

Comment:
What do we mean by "all with the same type" here?

Reply:
Every record in a RecordArray has the same type as each other.